.travis.yml

```diff
@@ -36,18 +36,15 @@ cache:
     - $HOME/.cache/pip
 
 before_install:
-  - rm -rf dist *.egg-info && python setup.py sdist
+    - rm -rf dist *.egg-info
 
 install: skip
 script:
-  - ci/cpp_test.sh
-  - ci/python_test.sh
+    - ci/cpp_test.sh
+    - ci/python_test.sh
 
 deploy:
+  provider: script
+  script: ci/python_sdist.sh
```

Comment:
I would use a filename that suggest this script has a side effect. sdist_to_pypi.sh or something like this.